content/en/docs/concepts/workloads/pods/pod-qos.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pod Quality of Service Classes
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+This page introduces _Quality of Service (QoS) classes_ in Kubernetes, and explains
+how Kubernetes assigns a QoS class to each Pods as a consequence of the resource
+constraints that you specify for the containers in that Pod. Kubernetes relies on this
+classification to make decisions about which Pods to evict when there are not enough
+available resources on a Node.
+
+<!-- body -->
+
+## Quality of Service classes
+
+Kubernetes classifies the Pods that you run and allocates each Pod into a specific
+_quality of service (QoS) class_. Kubernetes uses that classification to influence how different
+pods are handled. Kubernetes does this classification based on the
+[resource requests](/docs/concepts/configuration/manage-resources-containers/)
+of the {{< glossary_tooltip text="Containers" term_id="container" >}} in that Pod, along with
+how those requests relate to resource limits.
+This is known as {{< glossary_tooltip text="Quality of Service" term_id="qos-class" >}}
+(QoS) class. Kubernetes assigns every Pod a QoS class based on the resource requests
+and limits of its component Containers. QoS classes are used by Kubernetes to decide
+which Pods to evict from a Node experiencing
+[Node Pressure](/docs/concepts/scheduling-eviction/node-pressure-eviction/). The possible
+QoS classes are `Guaranteed`, `Burstable`, and `BestEffort`. When a Node runs out of resources,
+Kubernetes will first evict `BestEffort` Pods running on that Node, followed by `Burstable` and
+finally `Guaranteed` Pods. When this eviction is due to resource pressure, only Pods exceeding
+resource requests are candidates for eviction.
+
+### Guaranteed
+
+Pods that are `Guaranteed` have the strictest resource limits and are least likely
+to face eviction. They are guaranteed not to be killed until they exceed their limits
+or there are no lower-priority Pods that can be preempted from the Node. They may
+not acquire resources beyond their specified limits. These Pods can also make
+use of exclusive CPUs using the
+[`static`](/docs/tasks/administer-cluster/cpu-management-policies/#static-policy) CPU management policy.
+
+#### Criteria
+
+For a Pod to be given a QoS class of `Guaranteed`:
+
+* Every Container in the Pod must have a memory limit and a memory request.
+* For every Container in the Pod, the memory limit must equal the memory request.
+* Every Container in the Pod must have a CPU limit and a CPU request.
+* For every Container in the Pod, the CPU limit must equal the CPU request.
```

Comment:
```suggestion
* Every container in the Pod must have a memory limit and a memory request.
* For every container in the Pod, the memory limit must equal the memory request.
* Every container in the Pod must have a CPU limit and a CPU request.
* For every container in the Pod, the CPU limit must equal the CPU request.
```
nit: AFAIK, container is not an API object and should not be in upper camel case

Reply:
The way we capitalize container is contexts like this a bit of an open question. It's `Container` in parts of the OpenAPI. That said, I'm fine with the English word “container” which fits perfectly here without bringing APIs and serializations into the discussion.